pandas/_libs/tslibs/timestamps.pyi

```diff
@@ -0,0 +1,109 @@
+# flake8: noqa
+
+from typing import Any
+
+class RoundTo:
+    @property
+    def MINUS_INFTY(self) -> int: ...
+    @property
+    def PLUS_INFTY(self) -> int: ...
+    @property
+    def NEAREST_HALF_EVEN(self) -> int: ...
+    @property
+    def NEAREST_HALF_PLUS_INFTY(self) -> int: ...
+    @property
+    def NEAREST_HALF_MINUS_INFTY(self) -> int: ...
+
+def round_nsint64(values, mode, freq): ...
+
+class Timestamp:
+    value: Any = ...
```

Comment:
see https://github.com/pandas-dev/pandas/pull/28195#issuecomment-525845167

Reply:
not sure i follow. value is Any.

I want to avoid discussions about types in skeleton stubs in order to avoid hampering progress towards the resolution of #28133.

we are using `ignore_missing_imports=True` in our global config. mypy states "We recommend using this approach only as a last resort: it’s equivalent to adding a # type: ignore to all unresolved imports in your codebase." 

so will discuss types in a follow up, but I don''t see an issue with int for return types, maybe we will need to type the arguments differently for complete type safety.